src/__mocks__/Parser.ts

```diff
@@ -0,0 +1,4 @@
+jest.mock('parser/core/Parser')
```

Comment:
Same as the previous comment - this approach removes the need for mocking out the Meta/Report/Fight/Actor dependencies of the parser, which I don't really want to have to hoist since the rest of the mocking approach is not relying on those behaviors of the Parser.